README.md

```diff
@@ -1,2 +1,7 @@
 # urlimport
 A Go tool to download files from Http(s) and (s)ftp sites.
+- Support for Http(s) and (S)Ftp schemes
```

Comment:
Ok So both `sftp://demo:password@test.rebex.net:990/pub/example/readme.txt` and `sftp://demo:password@test.rebex.net:990/pub/example/readme.txt` are marked as error in UI itself in current DE url import . DE UI validates for `http(s) and ftp`. So I think this program will support what DE is currently supporting from the UI. I will update the `README.md` file.